Userland/Libraries/LibGL/Texture.cpp

```diff
@@ -143,6 +143,36 @@ void GLContext::gl_gen_textures(GLsizei n, GLuint* textures)
     }
 }
 
+void GLContext::gl_get_tex_image(GLenum target, GLint level, GLenum format, GLenum type, void* pixels)
+{
+    RETURN_WITH_ERROR_IF(level < 0 || level > Texture2D::LOG2_MAX_TEXTURE_SIZE, GL_INVALID_VALUE);
+    auto pixel_type_or_error = get_validated_pixel_type(target, GL_NONE, format, type);
+    RETURN_WITH_ERROR_IF(pixel_type_or_error.is_error(), pixel_type_or_error.release_error().code());
+
+    auto texture_2d = m_active_texture_unit->texture_2d_target_texture();
+    VERIFY(!texture_2d.is_null());
```

Comment:
Just to clarify, this should never fail because the default texture object is bound when `glBindTexture()` wasn't called or was called with argument `0`, correct?

Reply:
Yes, that's exactly why!